packages/vaadin-lumo-styles/gulpfile.js

```diff
@@ -24,6 +24,43 @@ function sortIconFilesNormalized(file1, file2) {
   return file1.replace(/-/g, '~').localeCompare(file2.replace(/-/g, '~'), 'en-US');
 }
 
+function createCopyright() {
+  return `/**
+ * @license
+ * Copyright (c) 2021 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */`;
+}
+
+function createIconset(folder, filenames, idPrefix = '') {
+  let output = `<svg xmlns="http://www.w3.org/2000/svg">\n<defs>\n`;
+  filenames.forEach(function (filename) {
+    // Skip non-svg files
+    if (filename.indexOf('.svg') === -1) {
+      return;
+    }
+
+    const content = fs.readFileSync(folder + filename, 'utf-8');
+    const path = content.match(/<path d="([^"]*)"/);
+    if (path) {
+      // var xScale = Math.min(1, fontHeight / glyphWidth);
```

Comment:
nit: Is this commented-out code still useful? Should it be removed?

Reply:
Thanks, removed.